README.md

```diff
@@ -11,6 +11,7 @@ Get started by navigating to one of the supported demos below.
 |                                          Demos | Description                                              | Details                            |
 |-----------------------------------------------:|:---------------------------------------------------------|:-----------------------------------|
 |         [`Checkout Example`](checkout-example) | E-commerce checkout flow with different payment methods. | [See below](#checkout-example)     | 
+|        [`Gift Card Example`](giftcard-example) | Gift Cards checkout flow using partial orders.           | [See below](#giftcard-example)     | 
```

Comment:
Link invalid: [See below(#giftcard-example)] should be: [See below(#gift-card-example)]